docs/en/status-communities/token-based-access-to-communities-and-channels.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 653
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Token-based access to communities and channels
+
+!!! tip ""
+    Currently, you can only configure token-based access using Status desktop.
+
+In the Status app, community [owners and admins][permissions-by-role-in-status-communities] can use token-based permissions to organize and control their communities and channels.
+
+As a community owner, you can configure token-based permission in [the community as a whole][set-up-your-community-permissions] and individual [channels within it][set-up-channel-permissions]. This functionality allows you to set up open communities with specific token-gated channels or [entirely closed communities][set-up-a-private-community] that require certain tokens for access.
+
+!!! note
+    Both community owners and admins can define token-based permission using Status desktop. For more information, check out [Permissions by role in Status Communities][permissions-by-role-in-status-communities].
+
+## Define and assign token-based access
+
+To define different access levels in your community, you start with a token.
+
+- [x] Choose one or more tokens you want to use. This can be an existing token (such as ETH, SNT or an ENS name), [a new token you mint][mint-tokens-for-your-community] or [a token you import][import-tokens-to-your-community] into the community.
+- [x] Create a new [community-level][set-up-your-community-permissions] or [channel-level][set-up-channel-permissions] permission using your the token.
+- [x] You can now token-gate access to your community or channels within it. For ideas on how to use these token-based permissions, check out [Using token-based access in your community](#using-token-based-access-in-your-community).
+
+## Using token-based access in your community
+
+Let's explore some examples of using token-based access in the Status app.
+
+### Provide early access to content
+
+As a content creator, you can set up a Status community for your fans. By [minting][mint-tokens-for-your-community] and [airdropping tokens][how-to-airdrop-tokens-in-status], you can give certain fans [access to an exclusive channel][grant-exclusive-access-with-tokens] where you release content before it's available elsewhere, adding value to your community.
+
+### Community access and administration
+
+Token-based permissions in Status Communities allow community owners to restrict access and delegate administrative functions to specific token holders. For more information, check out [Delegate admin functions to community members][delegate-admin-functions-to-community-members].
```

Comment:
'Token-based permissions in Status Communities allow community owners to restrict access and <...>' - again, I would avoid centring sentences around features. How about 'As a community owner, you can use token-based permissions to restrict access and <...>'?